### ISSUE TITLE
Calling get method of a map store inside onMount leads to infinet loop

### ISSUE THREAD

**Issue by blikblum** [nanostores/nanostores#127]
I get `RangeError: Maximum call stack size exceeded` with the code below

```js
import { map, action, onMount } from 'nanostores'

const initialValue = {}

export const appConfig = map(initialValue)

onMount(appConfig, async () => {
  if (appConfig.get() === initialValue) {
     const content =  await loadConfigFile()
     appConfig.set(JSON.parse(content))    
  }
})
```

**Comment by ai** [comment]
Will look today or tomorrow.

**Comment by ai** [comment]
Fixed 0a91822

**Comment by ai** [comment]
The fix was released in 0.7.1
